app/jobs/sync/zendesk/ticket_comments_to_notes.rb

```diff
@@ -0,0 +1,50 @@
+class Sync::Zendesk::TicketCommentsToNotes
+  include Sidekiq::Worker
+
+  # https://developer.zendesk.com/rest_api/docs/support/ticket_comments#list-comments
+  def perform(zendesk_ticket_id, page_number = 0)
+    require 'zendesk_api'
```

Comment:
It might make more sense to move the client initialization to the `config/initializers` folder. What are your thoughts?